lib/fog/compute/xen_server/real.rb

```diff
@@ -44,7 +44,31 @@ def connection_host
         def event
           @event ||= Fog::Compute::XenServer::Models::Event.new(service: self)
         end
+
+        private
+
+        def load_data(keep_type = nil, delete_templates = false)
```

Comment:
Since this method is for `servers` collection only maybe you want to rename it to `load_servers`?